pull test failed

Recent tests failed with the following

```
TOTAL                            3258    920    72%
Required test coverage of 49.49% reached. Total coverage: 71.76%
=========================== short test summary info ============================
FAILED tests/test_web.py::test_get_predict_invalid_season - assert 200 == 500
 +  where 200 = <Response [200 OK]>.status_code
============ 1 failed, 184 passed, 5 warnings in 660.33s (0:11:00) =============
make: *** [Makefile:21: test] Error 1
Error: Process completed with exit code 2.
```

--- a/f1pred/predict.py
+++ b/f1pred/predict.py
@@ -32,7 +32,12 @@ logger = get_logger(__name__)
 
 
 def resolve_event(jc: JolpicaClient, season: Optional[str], rnd: str) -> Tuple[int, int, Dict[str, Any]]:
-    """Resolve season/round to use. Never raises; falls back to current/last if needed."""
+    """Resolve season/round to use. Falls back to current/last if needed."""
+    # Explicitly validate inputs before the fallback try-block to catch fundamentally invalid requests
+    if season is not None and str(season).lower() != "current":
+        jc._validate_season(str(season))
+    jc._validate_round(str(rnd))
+
     try:
         if season is None or (isinstance(season, str) and season.lower() == "current"):
             if rnd == "next":

